### PR TITLE
initialise react app

### PR DESCRIPTION
**Pull request by ccobasi**
- Setup react and redux.
- Structure application using feature folder
- Add React Router 
- Display books and categories

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                <Link className="links" to="/categories">CATEGORIES</Link>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="card-body">
+          <div className="card">
+            <div className="card-body">
+              <div className="details">
+                <div className="row">
```
Comment (dasileker) [inline src/redux/books/books.js]: - [ ] The **`book.js`** should be a **`propTypes`** for the **Title** and **Author** see the screenshot 👇

hfhf

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              </div>
+              <div className="col">
+                <Link className="links" to="/categories">CATEGORIES</Link>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="card-body">
+          <div className="card">
+            <div className="card-body">
```
Comment (dasileker) [inline src/redux/categories/categories.js]: - [ ] Also the same should be for the **category**

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ccobasi** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **To Highlight**
- The PR is good.
- The commit are good. 
- The Linters are passing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+
+const Books = (props) => {
+  const { genre, title, author } = props;
+
+  return (
```
Comment (mikenath223) [inline src/redux/books/books.js:8]: - [ ] OPTIONAL - Kindly place the Book component in the components folder. The redux folder should only contain reducers and the redux store.

File: src/redux/categories/categories.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { PropTypes } from 'prop-types';
+
+const Categories = (props) => {
+  const { category } = props;
+  return (
+    <div className="main-container">
```
Comment (mikenath223) [inline src/redux/categories/categories.js:9]: - [ ] OPTIONAL - Also for the Category kindly place it in the components folder.

**Pull request merged**